Library/ViewModels/CommentTableViewFooterViewModelTests.swift

```diff
@@ -6,23 +6,27 @@ import XCTest
 final class CommentTableViewFooterViewModelTests: TestCase {
   private let vm: CommentTableViewFooterViewModelType = CommentTableViewFooterViewModel()
 
-  private let shouldStartLoaderIndicator = TestObserver<Bool, Never>()
+  private let shouldShowActivityIndicator = TestObserver<Bool, Never>()
 
   override func setUp() {
     super.setUp()
 
-    self.vm.outputs.shouldStartLoaderIndicator.observe(self.shouldStartLoaderIndicator.observer)
+    self.vm.outputs.shouldShowActivityIndicator.observe(self.shouldShowActivityIndicator.observer)
   }
 
-  func testOutputs() {
-    self.shouldStartLoaderIndicator.assertDidNotEmitValue()
+  func testShowActivityIndicator() {
```

Comment:
I know this is annoyingly specific but you should be following the conventions we already use in the project:

`test{OUTPUTNAME}_{VariableX}_{true/false}`

so in the case of these two tests it will be something like `testShouldShowActivityIndicator_Configure_True`

there's no set way to do it but we do follow this throughout the project

Reply:
Oh, Alright then.